compile_module.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+"""Compile Python module and returns its file name and base64-encoded bytecode
+as JSON.
+For use by 'run_with_batavia' script."""
+
+import argparse
+import json
+import py_compile
+import sys
+
+import base64
+import importlib
+import os
+import tempfile
+
+sys.path.insert(0, os.getcwd())
+
+
+def get_module_path(module):
+    if module.endswith('.py'):
```

Comment:
It seems weird to me that we have to special-case this but otherwise can use `importlib`? Why can't `importlib` be used in this case?

Reply:
I added this to allow someone to run a Python file as: ``./run_in_batavia.py /path/to/file.py`` where is where the execution will start. After that each time ``compile_module.py`` will be called it will be with an import path like ``package.module``. If importlib is given a full path to a Python file I don't think it will work. 

Although now that I think about it it should work if given ``file.py`` from the current directory. I will make this change and update the pull request.